pkgs/applications/networking/browsers/chromium/browser.nix

```diff
@@ -1,8 +1,47 @@
-{ stdenv, mkChromiumDerivation, channel }:
+{ stdenv, mkChromiumDerivation, channel, upstream-info, gcc, glib, nspr, nss, patchelfUnstable, enableWideVine }:
 
 with stdenv.lib;
 
-mkChromiumDerivation (base: rec {
+let
+  mkrpath = p: "${makeSearchPathOutput "lib" "lib64" p}:${makeLibraryPath p}";
+  widevine = stdenv.mkDerivation {
```

Comment:
Could use some suggestions about whether this is the best way to go about this. I've essentially copied the contents of the widevine plugin here.